{"0.1.0":{"info":{"author":"Jay Young(yjmade)","author_email":"dev@yjmade.net","bugtrack_url":null,"classifiers":["Development Status :: 4 - Beta","Intended Audience :: Developers","License :: OSI Approved :: ISC License (ISCL)","Programming Language :: Python :: 3","Programming Language :: Python :: 3.4","Programming Language :: Python :: 3.5"],"description_content_type":null,"docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/yjmade/django-errorlog","keywords":"django error log report","license":"MIT","maintainer":"","maintainer_email":"","name":"django-errorlog","package_url":"https://pypi.org/project/django-errorlog/","platform":"","project_url":"https://pypi.org/project/django-errorlog/","project_urls":{"Homepage":"https://github.com/yjmade/django-errorlog"},"provides_extra":null,"release_url":"https://pypi.org/project/django-errorlog/0.1.0/","requires_dist":null,"requires_python":"","summary":"Django reuseable app to collect the unexpcted exception then generate comprehansive report just like what you get in debug mode and store in database","version":"0.1.0","yanked":false,"yanked_reason":null},"last_serial":2500327,"urls":[{"comment_text":"","digests":{"blake2b_256":"8ce4cabece9795cf64eebb4e3e1f6863df9fbd74c0434ebb93bb1dfb467c6cc5","md5":"a9b471588b69e381801ea656a3cbf778","sha256":"be14b6d48b6d9545fa892c7536e1f25770e7649cc768b3f4144238b2db26078e"},"downloads":-1,"filename":"django-errorlog-0.1.0.tar.gz","has_sig":false,"md5_digest":"a9b471588b69e381801ea656a3cbf778","packagetype":"sdist","python_version":"source","requires_python":null,"size":7123,"upload_time":"2016-12-04T14:20:59","upload_time_iso_8601":"2016-12-04T14:20:59.919318Z","url":"https://files.pythonhosted.org/packages/8c/e4/cabece9795cf64eebb4e3e1f6863df9fbd74c0434ebb93bb1dfb467c6cc5/django-errorlog-0.1.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.1":{"info":{"author":"Jay Young(yjmade)","author_email":"dev@yjmade.net","bugtrack_url":null,"classifiers":["Development Status :: 4 - Beta","Intended Audience :: Developers","License :: OSI Approved :: ISC License (ISCL)","Programming Language :: Python :: 3","Programming Language :: Python :: 3.4","Programming Language :: Python :: 3.5"],"description_content_type":null,"docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/yjmade/django-errorlog","keywords":"django error log report","license":"MIT","maintainer":"","maintainer_email":"","name":"django-errorlog","package_url":"https://pypi.org/project/django-errorlog/","platform":"","project_url":"https://pypi.org/project/django-errorlog/","project_urls":{"Homepage":"https://github.com/yjmade/django-errorlog"},"provides_extra":null,"release_url":"https://pypi.org/project/django-errorlog/0.1.1/","requires_dist":null,"requires_python":"","summary":"Django reuseable app to collect the unexpcted exception then generate comprehansive report just like what you get in debug mode and store in database","version":"0.1.1","yanked":false,"yanked_reason":null},"last_serial":2500327,"urls":[{"comment_text":"","digests":{"blake2b_256":"ae084f9e97a2c3bcba20864cc43e78a3094b7364af004135b85d3c9da96911b8","md5":"58d3e8e71fee00c3640e1d80ad45d060","sha256":"c0c2c9a76f0c5406841b5125f210602969f4e5a993194617b01afee2a2b75c20"},"downloads":-1,"filename":"django-errorlog-0.1.1.tar.gz","has_sig":false,"md5_digest":"58d3e8e71fee00c3640e1d80ad45d060","packagetype":"sdist","python_version":"source","requires_python":null,"size":13257,"upload_time":"2016-12-05T13:53:14","upload_time_iso_8601":"2016-12-05T13:53:14.246279Z","url":"https://files.pythonhosted.org/packages/ae/08/4f9e97a2c3bcba20864cc43e78a3094b7364af004135b85d3c9da96911b8/django-errorlog-0.1.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}